### PR TITLE
use initialization protocol. fix: #104

### PR DESCRIPTION
**Pull request by vonbecmann**
use initialization protocol for instance size initialize method.

### CODE REVIEW

File: repository/Zinc-Character-Encoding-Core.package/ZnCrPortableWriteStream.class/properties.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"commentStamp" : "SvenVanCaekenberghe 11/7/2018 13:56",
+	"commentStamp" : "",
```
Comment (vonbecmann) [inline repository/Zinc-Character-Encoding-Core.package/ZnCrPortableWriteStream.class/properties.json:2]: Why?

File: repository/Zinc-Character-Encoding-Core.package/ZnNewLineWriterStream.class/properties.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"commentStamp" : "SvenVanCaekenberghe 11/7/2018 13:55",
+	"commentStamp" : "",
```
Comment (vonbecmann) [inline repository/Zinc-Character-Encoding-Core.package/ZnNewLineWriterStream.class/properties.json:2]: why?

File: repository/Zinc-HTTP.package/ZnLoggingReadStream.class/properties.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-	"commentStamp" : "SvenVanCaekenberghe 7/28/2021 11:14",
+	"commentStamp" : "",
```
Comment (vonbecmann) [inline repository/Zinc-HTTP.package/ZnLoggingReadStream.class/properties.json:2]: ??

**Comment by vonbecmann** [comment]
i don't like it. it changed many things unknown to me.

**Comment by vonbecmann** [comment]
sorry for the noise.

**Comment by svenvc** [comment]
I usually don't really care about those meta timestamps in properties.json as git is enough.

**Pull request closed without merging**